cobigen-eclipse/cobigen-eclipse-test/src/main/java/com/devonfw/cobigen/eclipse/test/AdaptTemplatesTest.java

```diff
@@ -31,71 +32,79 @@
-        project.getProject().refreshLocal(IResource.DEPTH_INFINITE, new NullProgressMonitor());
-        this.tmpMavenProjectRule.updateProject();
-
-        EclipseCobiGenUtils.runAndCaptureUpdateTemplates(bot);
-        EclipseCobiGenUtils.runAndCaptureAdaptTemplates(bot);
-        EclipseUtils.updateMavenProject(bot, ResourceConstants.CONFIG_PROJECT_NAME);
-
-        EclipseUtils.openErrorsTreeInProblemsView(bot);
-
-        // expand the new file in the package explorer
-        SWTBotView view = bot.viewById(JavaUI.ID_PACKAGES);
-        SWTBotTreeItem javaClassItem = view.bot().tree().expandNode(testProjName, "adapt-templates.yml");
-        javaClassItem.select();
-
-        IProject adaptedTemplatesProj = ResourcesPlugin.getWorkspace().getRoot()
-                .getProject(ConfigurationConstants.COBIGEN_TEMPLATES);
-
-        assertThat(adaptedTemplatesProj.exists()).isTrue();
-    }
-
-    /**
-     * Test of testBasicOpenAPIGenerationWithAdaptTemplates with custom COBIGEN_HOME
-     * environment variable
-     *
-     * @throws Exception test fails
-     */
-    @Test
-    public void testAdaptTemplates() throws Exception {
-
-        File tmpProject = this.tempFolder.newFolder("playground", "project", "templates");
-        withEnvironmentVariable(ConfigurationConstants.CONFIG_ENV_HOME, tmpProject.getParentFile().getAbsolutePath())
-                .execute(() -> testBasicOpenAPIGenerationWithAdaptTemplates());
-    }
+  /** Temporary files rule to create temporary folders or files */
+  @Rule
+  public TemporaryFolder tempFolder = new TemporaryFolder();
+
+  /** Root path of the Test Resources */
+  private static final String resourcesRootPath = "src/main/resources/OpenAPITest/";
+
+  /** Line separator, e.g. for windows '\r\n' */
+  public static final String LINE_SEPARATOR = java.lang.System.getProperty("line.separator");
+
+  /**
+   * Setup workbench appropriately for tests
+   *
+   * @throws Exception test fails
+   */
+  @BeforeClass
+  public static void setupClass() throws Exception {
+
+    EclipseUtils.cleanWorkspace(bot, true);
+  }
+
+  /**
+   * Test for external projects (not in workspace) taken as input for generation
+   *
+   * @throws Exception test fails
+   */
+  public void testBasicOpenAPIGenerationWithAdaptTemplates() throws Exception {
+
+    // copy sample project to external location and import it into the workspace
+    String testProjName = "ExtTestProj";
+    IJavaProject project = this.tmpMavenProjectRule.createProject(testProjName);
+    FileUtils.copyFile(new File(resourcesRootPath + "input/adapt-templates.yml"),
+        project.getUnderlyingResource().getLocation().append("adapt-templates.yml").toFile());
+    project.getProject().refreshLocal(IResource.DEPTH_INFINITE, new NullProgressMonitor());
+    this.tmpMavenProjectRule.updateProject();
+
+    EclipseCobiGenUtils.runAndCaptureUpdateTemplates(bot);
```

Comment:
Fill Templates Folder with two old jars instead

Reply:
RunAndCaptureUpdateTemplates ist downloading the old jars.

Reply:
We need to remove the update of the old templates as this should not be supported anymore.
Instead we should manually download the old templates and place these into the .cobigen/templates directory.
Background info: The whole update feature will later download the template sets instead of the old monolithic templates and can't be used for this test then.